pkg/commands/artifact/run.go

```diff
@@ -648,6 +662,7 @@ func initScannerConfig(opts flag.Options, cacheClient cache.Cache) (ScannerConfi
 				Full: opts.LicenseFull,
 			},
 		},
+		RemoteOption: remoteOption,
```

Comment:
It looks we duplicate `remoteOption` in [ArtifactOption](https://github.com/aquasecurity/trivy/pull/3906/files#diff-3b6c25b1435d1150f7492a6c5b47429c0f0bf1f630340582163cece9823168b7R650) and here.
Perhaps we can use `remoteOption` from `ArtifactOption`?

Reply:
Great feedback! Fixed in https://github.com/aquasecurity/trivy/pull/3906/commits/554e01c19f380c2374288cdfca138a80b6546ede